app/views/requests/create.js.erb

```diff
@@ -0,0 +1 @@
+$('#<%= @book_id %>').html('<p>Pending</p>');
```

Comment:
It's a good practice to not hardcode this sort of thing.

You can call `@request.status.titleize` and you are covered if in the future you want to rename the statuses.